### PR TITLE
PP-316: rif aggregation fix ci cd

### PR DESCRIPTION
**Pull request by kenny08gt**
# What
The CI (github actions) don't work properly, seems to fail due a timeout because never start.

# Why
To help pull request reviews and ensure to not bring breaking changes

# Ref
https://rsklabs.atlassian.net/browse/PP-316

### CODE REVIEW

**Comment by kenny08gt** [comment]
Right now the ci is failing but because the test fails (before was not even executed), fixing that it's out of the scope of [PP-316](https://rsklabs.atlassian.net/browse/PP-316 ). Probably related with [PP-142](https://rsklabs.atlassian.net/browse/PP-142).

**Comment by kenny08gt** [comment]
@antomor 
1. Actually all updates I made in `ci.yml` are in base on those two files (`ci-iov-integration.yml` and `ci-iov.yml`).
2. For the integration test, it changes the [automine conf ](https://github.com/rsksmart/ri-aggregation/pull/32/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR103).

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,31 +4,41 @@ on:
...
     steps:
       - uses: actions/checkout@v2
 
-      - name: setup-env
+      - name: modify rsk node conf and install mkcert
         run: |
-          echo ZKSYNC_HOME=$(pwd) >> $GITHUB_ENV
-          echo $(pwd)/bin >> $GITHUB_PATH
-
+          sed -i '/autoMine/ s/true/false/' docker/rskj/node.conf
```
Comment (antomor) [inline .github/workflows/ci.yml:18]: why do we need to set `automine` false  to run the lint job?

Comment (kenny08gt) [reply .github/workflows/ci.yml:18]: I think we don't need it, I just copy the configuration from ci-iov-integration.yml to make it run

**Pull request merged**